docs/how_to/noisy_simulators.rst

```diff
@@ -0,0 +1,430 @@
+    options.execution.shots = 1000
+    options.optimization_level = 0 # no optimization
+    options.resilience_level = 0 # no error mitigation
+
+Run the circuits on Sampler
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+We shall now sample the circuit to get the result probability
+distribution. We shall be utilizing the `Sampler
+primitive <https://qiskit.org/documentation/partners/qiskit_ibm_runtime/stubs/qiskit_ibm_runtime.Sampler.html>`__
+to do the same. To learn how to use the ``Sampler`` primitive and how to
+get started using Qiskit Runtime Sessions, you can check this tutorial:
+`Get started with the Sampler
+primitive <https://qiskit.org/documentation/partners/qiskit_ibm_runtime/tutorials/how-to-getting-started-with-sampler.html>`__.
+
+.. code-block:: python
+
+    with Session(service=service, backend=backend):
+        sampler = Sampler(options=options)
+        job = sampler.run(
+            circuits=[qc]*len(phases),
+            parameter_values=individual_phases
+        )
+        result = job.result()
+
+.. code-block:: python
+
+    import matplotlib.pyplot as plt
+
+    # the probablity of being in the 1 state for each of these values
+    prob_values = [dist.get(1, 0) for dist in result.quasi_dists]
+
+    plt.plot(phases, prob_values, 'o', label='Simulator')
+    plt.plot(phases, np.sin(phases/2,)**2, label='Theory')
+    plt.xlabel('Phase')
+    plt.ylabel('Probability')
+    plt.legend()
+
+
+
+
+.. parsed-literal::
+
+    <matplotlib.legend.Legend at 0x7f7fd233b6d0>
+
+
+
+
+.. image:: /docs/images/noisy-sim-sampler-ideal.png
+
+
+Run the circuits on Estimator
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To learn how to start a session for Estimator, you may check this
+tutorial: `Get started with the Estimator
+primitive <https://qiskit.org/documentation/partners/qiskit_ibm_runtime/tutorials/how-to-getting-started-with-estimator.html>`__.
+
+The Estimator will bind single-qubit rotations to get hamiltonians
+before it returns expectation values of quantum operators. Therefore,
+the circuit doesn’t require any measurements. Currently the circuit
+``qc`` has measurements so we will remove these with
+``remove_final_measurements``.
+
+.. code-block:: python
+
+    qc_no_meas = qc.remove_final_measurements(inplace=False)
+    qc_no_meas.draw('mpl')
+
+
+
+
+.. image:: /docs/images/noisy-sim-estimator-circuit.png
+
+
+
+.. code-block:: python
+
+    from qiskit.quantum_info import SparsePauliOp
+
+    ZZ = SparsePauliOp.from_list([("ZZ", 1)])
+    print(f"  > Observable: {ZZ.paulis}")
+
+
+.. parsed-literal::
+
+      > Observable: ['ZZ']
+
+
+With this observable, the expectation value is calculated by the
+equation below.
+
+.. math::
+
+
+   \langle ZZ\rangle =\langle \psi | ZZ | \psi\rangle=\langle \psi|(|0\rangle\langle 0| -|1\rangle\langle 1|)\otimes(|0\rangle\langle 0| - |1\rangle\langle 1|) |\psi\rangle =|\langle 00|\psi\rangle|^2 - |\langle 01 | \psi\rangle|^2 - |\langle 10 | \psi\rangle|^2 + |\langle 11|\psi\rangle|^2
```

Comment:
This is not rendering correctly in the preview. I _think_ because there's an extra blank line after `.. math::`.

Reply:
I removed one extra line, but on my side, it is still not rendering correctly in the preview.

Reply:
I believe this will render correctly when deployed on the qiskit docs, just not in the github rst preview